src/tests/integration/test-data/show/ArithmeticCallTest.test_double_add(uint256,uint256).cse.expected

```diff
@@ -202,7 +202,9 @@ module SUMMARY-TEST%ARITHMETICCALLTEST.TEST-DOUBLE-ADD(UINT256,UINT256):0
                          0
                        </balance>
                        <storage>
-                         ( 27 |-> 491460923342184218035706888008750043977755113263 )
+                         ( ( 11 |-> 1 )
+                         ( ( 27 |-> 491460923342184218035706888008750043977755113263 )
+                         ( 7 |-> 1 ) ) )
```

Comment:
What do these two new storage slots correspond to?

Reply:
They are introduced by running the (implicit) constructor since `test_foundry_dependency_automated` now uses `run_constructor=True`. This contract inherits from `Test`, which inherits from `DSTest`, which has a member
```
    bool public IS_TEST = true;
```
and `StdChains`, which has a member
```
    bool private fallbackToDefaultRpcUrls = true;
```
I'm pretty sure the 1s are these getting initialized.

Reply:
Excellent, thank you.